Model/SepaDirectDebitPaymentMethod.php

```diff
@@ -257,4 +260,19 @@ private function _isRefundAction($sAction)
     {
         return $sAction === TransactionModel::ACTION_CREDIT;
     }
+
+    /**
+     * @inheritdoc
+     * @throws InputException
+     *
+     * @since 1.1.0
+     */
+    public function onBeforeTransactionCreation()
+    {
+        $oConfig = Registry::getConfig();
+
+        if (!$oConfig->getRequestParameter('wdsepadd_checkbox')) {
+            throw new InputException('Mandate information was not accepted.');
```

Comment:
Should that maybe be multi-language? I guess the error message is displayed in the frontend as it is passed to `OrderHelper::setSessionPaymentError`

Reply:
Please see the note in the PR description. tldr: It's not translated because it's only displayed to users who try to hack the system.

Reply:
👍